doc/programming_guide/asrc/overview.rst

```diff
@@ -2,11 +2,32 @@
 Overview
 ********
 
+.. warning::
+
+   This example is based on the RTOS framework and drivers. This choice allowed simplifying the example design, but it lead to some latency in the system.
```

Comment:
`This choice allowed simplifying the example design, but it lead to some latency in the system.` -> `This choice simplifies the example design, but it leads to high latency in the system.`

Reply:
Fixed in https://github.com/xmos/sln_voice/pull/365/commits/86b87358a4a8763ecd3426194ce91bcc9141c5fa.